docs/api.md

```diff
@@ -292,6 +293,23 @@ await chromeless.scrollTo(500, 0)
 
 ---------------------------------------
 
+<a name="api-scrolltoelement" />
+
+### scrollToElement(selector: string): Chromeless<T>
+
+Scroll to location of element.
```

Comment:
Might add a note that the expected behavior is the element will be at the top of viewport just to remove that uncertainty